core/src/main/java/io/aiven/klaw/helpers/db/rdbms/DeleteDataJdbc.java

```diff
@@ -248,10 +248,14 @@ public String deleteAclSubscriptionRequest(int req_no, int tenantId) {
   }
 
   @Transactional
-  public void deleteTopics(Topic topic) {
+  public CRUDResponse<Topic> deleteTopics(Topic topic) {
     log.debug("deleteTopics {}", topic.getTopicname());
     topicRepo.deleteByTopicnameAndEnvironmentAndTenantId(
         topic.getTopicname(), topic.getEnvironment(), topic.getTenantId());
+    return CRUDResponse.<Topic>builder()
+        .resultStatus(ApiResultStatus.SUCCESS.value)
+        .entities(List.of(topic))
+        .build();
   }
```

Comment:
@snuyanzin I have made some updates here where to update the cache properly we now can send back a crud response with the entities so after updates are committed to the DB they can be updated in the cache. You were working in this area over the last week so your input would be great!

Reply:
I didn't check it functionality, however if it works then should be ok.

nit: Can we replace
```java
CRUDResponse.<Topic>builder()
        .resultStatus(ApiResultStatus.SUCCESS.value)
        .entities(List.of(topic))
        .build();
```
with
```java
CRUDResponse.ok(List.of(topic));
```
In a similar way it was done for responses?

Reply:
I was thinking the same thing actually will take a look at that